src/mixins/subscriptionView.js

```diff
@@ -36,10 +36,5 @@ export default {
     next(vm => {
       vm.$workflowService.startSubscriptions()
     })
-  },
-  beforeRouteUpdate (to, from, next) {
-    next(vm => {
-      vm.$workflowService.startSubscriptions()
-    })
```

Comment:
Need to test this one very carefully.

Reply:
From my description in https://github.com/cylc/cylc-ui/pull/1186/commits/cc68f90eaa5e30c7e6361cf6381cb49ed06f8359

> Passing a callback to `next()` is only supported in `beforeRouterEnter()`, not `beforeRouterUpdate()`, so that was not doing anything this whole time.
> See https://v3.router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards

Reply:
> specifically covered by a unit test

The problem with that test was that it was passing in a function to the `next` arg that called the arg provided to that function, but vue-router does not necessarily do this - presumably it just passes in a function that is called without args.

Reply:
From investigating #1248, this will have to be brought back in the correct form as
```suggestion
  },
  beforeRouteUpdate () {
    this.$workflowService.startSubscriptions()
  }
```
(however this is still broken until #1248 is fixed)